src/cascadia/TerminalSettingsModel/ActionMapSerialization.cpp

```diff
@@ -49,6 +56,30 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         return warnings;
     }
 
+    Json::Value ActionMap::ToJson() const
+    {
+        Json::Value actionList{ Json::ValueType::arrayValue };
+        for (const auto& [_, cmd] : _ActionMap)
+        {
+            const auto cmdImpl{ winrt::get_self<implementation::Command>(cmd) };
+            const auto& cmdJsonArray{ cmdImpl->ToJson() };

```

Comment:
what? a command serializes to multiple instances?? why? How do we validate that contract? It's just called `ToJson` :|

Reply:
Added the following. Hope it clears it up. :)
```c++
// Command serializes to an array of JSON objects.
            // This is because a Command may have multiple key chords associated with it.
            // The name and icon are only serialized in the first object.
            // Example:
            // { "name": "Custom Copy", "command": "copy", "keys": "ctrl+c" }
            // {                        "command": "copy", "keys": "ctrl+shift+c" }
            // {                        "command": "copy", "keys": "ctrl+ins" }
```

Reply:
There's a serialization test that covers this case:
```js
{
	[
		{ "command": { "action": "adjustFontSize", "delta": 1 }, "keys": "ctrl+c" },
		{ "command": { "action": "adjustFontSize", "delta": 1 }, "keys": "ctrl+d" }
	]
}
```